app/models/project.rb

```diff
@@ -618,20 +618,16 @@ def check_status(deprecated_or_removed = false)
       update_attribute(:status, "Removed")
     elsif platform.downcase != "packagist" && [400, 404, 410].include?(response.response_code)
       update_attribute(:status, "Removed")
-    elsif platform.downcase == "clojars" && response.response_code == 404
-      update_attribute(:status, "Removed")
-    elsif platform.downcase == "pypi" && response.response_code == 404
-      update_attribute(:status, "Removed")
     elsif can_have_entire_package_deprecated?
       result = platform_class.deprecation_info(self)
       if result[:is_deprecated]
         update_attribute(:status, "Deprecated")
         update_attribute(:deprecation_reason, result[:message])
       else # in case package was accidentally marked as deprecated (their logic or ours), mark it as not deprecated
-        update_attribute(:status, '')
+        update_attribute(:status, nil)
```

Comment:
It felt incorrect to me that we had two cases of unsetting the status, one where it was an empty string and the other where it was nil. I changed this case to nil as well, and updated the specs accordingly. If this is incorrect then let me know and I can revert this change.

Reply:
looks like the "" statuses are a very small number, so we could probably update those to nil after deploy?